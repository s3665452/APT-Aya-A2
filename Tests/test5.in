RLYBRLBRRLUUURLBYBRULLLURUUBRRRBYYRUULRURYRYLURYYRYRLLURYBLBLRBLLLUBYLYYUYYBBYLUYYBBUYLYBUBRBLUBBUBU
Sean
Mikee
turn 1 R 1
turn 3 U 2
turn 0 L 2
turn 2 R 3
turn 0 B 3
turn 5 B 4