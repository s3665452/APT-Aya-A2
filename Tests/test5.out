Factories:
0: Y R L Y R U
1: 
2: 
3: 
4: U R L B 
5: 

Score for Sean: 0
Mosaic for Sean:
1:         R || . . . . .
2:       L L || . . . . .
3:     . B B || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken:

Score for Mikee: 0
Mosaic for Mikee:
1:         . || . . . . .
2:       U U || . . . . .
3:     . R R || . . . . .
4:   . . . B || . . . . .
5: . . . . . || . . . . .
broken: F


